docs/styling/common-props.md

```diff
@@ -0,0 +1,241 @@
+# Style and Layout Props
+
+```python exec
+import reflex as rx
+
+props = {
+    "align": {
```

Comment:
`align` is the radix variant of the prop; i wonder if we should recommend the actual css prop `align_items`

Reply:
this is a good point. I do mention that align is equivalent to align_items in the text description